includes/class-amp-travel-blocks.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * AMP Travel blocks class.
+ *
+ * @package WPAMPTheme
+ */
+
+/**
+ * Class AMP_Travel_Blocks.
+ *
+ * @package WPAMPTheme
+ */
+class AMP_Travel_Blocks {
+
+	/**
+	 * AMP_Travel_Blocks constructor.
+	 */
+	public function __construct() {
+		if ( function_exists( 'gutenberg_init' ) ) {
```

Comment:
@miina, could you please create an `init()` method, and move the `add_action()` and `add_filter()` calls inside it?